src/controllers/signin.js

```diff
@@ -0,0 +1,35 @@
+/* login */
+  static login(req, res) {
+    // Validate inputs
+    let checkInput = false;
+    checkInput = Validate.email(req.body.email, true);
+    if (checkInput.isValid === false) {
+      return res.status(400).json({
+        status: 400,
+        error: checkInput.error,
+      });
+    }
+    let isUser = {};
+    users.forEach((user) => {
+      if (user.email === req.body.email && user.password === req.body.password) {
+        isUser = {
```

Comment:
@smugabekazi, as said on the slack channel, please do not return user details in your signup and sign in endpoints. just return the token.